core/server/analytics.js

```diff
@@ -155,7 +137,12 @@ Meteor.methods({
     });
 
     const user = Meteor.user();
-    analytics.track(userId, '🎤 Conference Attend', { level_id: user.profile.levelId, zone_id: traits.zoneId, zone_name: traits.zoneName });
+    analytics.track(userId, '🎤 Conference Attend', {
+      guild_id: user.guildId,
```

Comment:
Why tracking the `guild_id` here and in other similar events? It seems to me that it is the kind of information that belongs to the user and thus to `identify` calls. :)

Reply:
From what I've seen in the documentation, adding the attribute of a group to an event allows to link an event to a group (in addition to linking it to an user)
<img width="1652" alt="Capture d’écran, le 2022-09-01 à 17 30 01" src="https://user-images.githubusercontent.com/3781087/187953415-41d3db80-3b38-4b44-be00-cf1718ec4a0b.png">

Reply:
Okay, weird but right, seems to be the Mixpanel way indeed. TIL!

Reply:
I have the feeling that a tool like June is smart enough to automatically make the link between event -> user -> group, while tools like mixpannel don't make the link between the data, at least that's what I noticed with my tests